### ISSUE TITLE
yarn link results in Unexpected value 'MyModule' imported by the module 'AppModule'. Please add a @NgModule annotation.

### ISSUE THREAD

**Issue by Geoffrey0** [dherges/ng-packagr#132]
## Type of Issue

- [X] Bug Report
- [ ] Feature Request

## Description

When linking the dist folder created by 'ng-packagr -p ng-package.json'. It throws a runtime error 'Unexpected value 'MyModule' imported by the module 'AppModule'. Please add a @NgModule annotation.' When publishing the same dist folder to npm and installing it, it does not give any errors.

Copying the files from the dist folder to the apps node modules works but this is not a solution that we want to pursue.

#### How To Reproduce

Run 'ng-packagr -p ng-package.json'. Link the dist folder by doing yarn link. Now go to the app that want to use the library and use yarn link my-library. 

#### Expected Behaviour

Expected behaviour yarn link would give the same results as publishing to npm repo (for local development). 

#### Version Information

```
ng-packagr: v1.2.0
node: v7.10.0
@angular: v4.2.4
rxjs: 5.4.2
zone.js: 0.8.14
```

**Comment by Geoffrey0** [comment]
@matt328 Thanks for the answer. This indeed works 👍 Angular CLI should really add this to their documentation.

**Comment by deebloo** [comment]
yeah the current cli story for using linked libs doesn't mention the flag.
